spec/lib/mcb/commands/courses/touch_spec.rb

```diff
@@ -13,13 +13,8 @@
     new_provider
   end
 
-  let(:course) { create :course, updated_at: 1.day.ago, changed_at: 1.day.ago, provider: provider }
-  let(:rolled_over_course) do
-    new_course = course.dup
-    new_course.update(provider: rolled_over_provider)
-    new_course.save
-    new_course
-  end
+  let(:course) { create :course, updated_at: 1.day.ago, changed_at: 1.day.ago, provider: provider, applications_open_from: DateTime.new(provider.recruitment_cycle.year.to_i, 1, 1) }
```

Comment:
wonder should that go into factory `applications_open_from`

Reply:
like that
```ruby
factory :course do
  applications_open_from { DateTime.new(provider.recruitment_cycle.year.to_i, 1, 1) } }
```

Reply:
Have added a more thorough random sampled date to the factory. This did exist in the factory before btw. I'll change the test to be + 1 day though 👍